.github/workflows/ci.yml

```
@@ -359,3 +359,7 @@ jobs:
     - name: Run tests
       working-directory: ./debug_utils/sierra-emu
       run: make test
+    - name: Run corelib's tests
```

I would add here something to make more visible that this job is in progress. Maybe:

```suggestion
    - name: Run corelib's tests (WIP)
```

---

Yes I agree, done.

---

It seems [this commit](https://github.com/lambdaclass/cairo_native/commit/48fb545ea0021f6235f669811173037568d7ad55) reverted the suggested change.

---

Oh my bad. Changing it.